external-providers/java-external-provider/pkg/java_external_provider/provider.go

```diff
@@ -228,6 +228,16 @@ func (p *javaProvider) Init(ctx context.Context, log logr.Logger, config provide
 	}
 	log = log.WithValues("provider", "java")
 
+	if config.RPC != nil {
+		return &javaServiceClient{
+			rpc:               config.RPC,
```

Comment:
What will this config look like? A sample somewhere could be useful

Reply:
It is only for someone using this as library and calling java-external-provider from go code. I expect the use case to be limited to us (kai/kantra) and I don't expect it to be a user facing change. 

Thoughts on where/how this should be documented in that case?

Reply:
If that's the case then it's cool :+1: